Sources/PostgresNIO/Connection/PostgresConnection.swift

```diff
@@ -377,6 +387,7 @@ extension PostgresConnection {
         to socketAddress: SocketAddress,
         tlsConfiguration: TLSConfiguration? = nil,
         serverHostname: String? = nil,
+        requireBackendKeyData: Bool = true,
```

Comment:
This would be a breaking change. For now, let's not add the option here. Users who want to the new feature, shall use async await API, or create a separate PR.